features/_your_results/period_percent_table.feature

```diff
@@ -0,0 +1,9 @@
+Feature:
+  As a user who has just completed the WPCC tool, I should be able to see helpful information on how the yearly increments is done. 
+
+  Scenario: Display table of percentages for each period
+    Given I am on the YourDetailsPage
+    When I fill in my details
+    And I click the Next button
+    And I move on to the results page
+    Then I should see a table with a column heading for each period
```

Comment:
@margOnline : I'm failing to understand what's the confusion? There is this table, which is static table with some numbers, has to be displayed when user clicks the link. If you have other solution in mind, please share?